Readme.md

```diff
@@ -363,6 +370,17 @@ Notice how the command the sub-command and flag info are displayed.
 
 Read more about the [help message](http://docs.getguaka.com/Protocols/HelpGenerator.html)
 
+## Cross-Platform utility libraries aka batteries
+
+Writing a command line application is more than just parsing the command line arguments and flags. 
+
+Swift ecosystem is still very young e.g no cross platform stdlib and we did not wanted to make Guaka depend on libFoundation, so we rolled up our sleeves and made these babies cross-platform swift libraries so you can be productive instantaneously:
```

Comment:
"a few small cross-platform Swift libraries" instead of "these babies cross-platform swift libraries"